source/includes/_capture_external_event_api.md

```diff
@@ -0,0 +1,72 @@
+# Capture External Event API
+
+Capture External Event API is a public ReachLocal external interface for vendors to use.
+
+## Authorization - Capture External Event API
+
+All requests must include an [Authorization](#authentication) header.
+
+The email address used to obtain the OAuth access token must be associated with the GMAID of the requested advertiser.  If it is not, the API will return an HTTP response with status code 403 and the following body:
+
+`{ "message": "Forbidden" }`
+
+## Create Chat Event
+
+This endpoint is used to create a chat event.
+
+### HTTP Request
+
+| Method | URI Format |
+|---|---|
+| POST | https://externalapi.reachlocal.com/api/chats |
+
+### Post Body Parameters
+
+Parameter | Required | Datatype | Nullable | Description
+--------- | -------- | -------- | -------- | -----------
+eventTime | yes  | string | no | The date/time in UTC that the chat started.
+advertiserId | yes*  | string | no | The global master advertiser id (for example, USA_142687).
+scopeId | yes*  | string | no | Legacy use only. The global master advertiser id (for example, USA_142687).
+data | yes | string | no | A JSON hash representing the attributes for the chat. See the following section for detail.
+
+\* Either advertiserId or scopeId must be present. 
+
+
+This section defines the data parameter JSON hash.
+
+Data Field | Required | Datatype | Nullable | Description
+--------- | -------- | -------- | -------- | -----------
+provider | yes | string | no | The name of the service provider i.e. Nanorep.
+visitId | yes | string | no | The Reach Local visitId of the chat user.
+chatDuration | yes | number | yes | The duration of the chat in seconds.
+chatSummary | yes | string | no | A summary of the chat.
+chatTextFull | yes | string | no | The full text of the chat.
+chatVisitorEmail |yes | string | no | The email address of the visitor.
+chatVisitorName | yes | string | no | The name of the chat visitor.
+chatVisitorPhone | yes | string | no | The phone number for this chat visitor.
+
+### Examples:
```

Comment:
You included an examples section, but no example code, only an example body.  Please include example code in both shell (curl), ruby and Java.  The secret to creating these is using Postman.  It generates it for you.

Reply:
What does your API return on a successful call?  Think about what the HTTP response code is as well as the response body.  What does your API return when provided invalid input?

Reply:
Yeah, Eric wanted me to get the spec out there wuickly so the nonorep folks could see the format of the Chat event. I agree we need to add this but I hoping to capture this after I complete development which is still ongoing but hopefully finished soon.